README.md

```diff
@@ -2,9 +2,9 @@
 
 go-github is a Go client library for accessing the [GitHub API][].
 
-**Documentation:** [![GoDoc](https://godoc.org/github.com/google/go-github/github?status.svg)](https://godoc.org/github.com/google/go-github/github)  
-**Mailing List:** [go-github@googlegroups.com](https://groups.google.com/group/go-github)  
-**Build Status:** [![Build Status](https://travis-ci.org/google/go-github.svg?branch=master)](https://travis-ci.org/google/go-github)  
+**Documentation:** [![GoDoc](https://godoc.org/github.com/google/go-github/github?status.svg)](https://godoc.org/github.com/google/go-github/github)
+**Mailing List:** [go-github@googlegroups.com](https://groups.google.com/group/go-github)
+**Build Status:** [![Build Status](https://travis-ci.org/google/go-github.svg?branch=master)](https://travis-ci.org/google/go-github)
```

Comment:
You should undo these changes. I'm guessing your editor did them for you.

The two spaces at the end of those lines are significant, they mark a line break after each line.

Compare what the README looks like now:

![image](https://cloud.githubusercontent.com/assets/1924134/21071513/a1f0e46c-be70-11e6-8a66-6b35ee494b53.png)

And after this change, it's less readable:

![image](https://cloud.githubusercontent.com/assets/1924134/21071515/aee9a53c-be70-11e6-82e3-481fb674346d.png)